docs/operations/control_plane_migration.md

```diff
@@ -24,11 +24,38 @@ The etcd backups will be copied over to the `BackupBucket` of the `Destination S
+>
+> The nodes of your `Shoot` cluster must have network connectivity to the `Shoot`'s `kube-apiserver` and the `vpn-seed-server` once they are migrated to the `Destination Seed`. Otherwise, the `Restore` operation will get stuck at the `Waiting until the Kubernetes API server can connect to the Shoot workers` step. However, if you do end up in this case and cannot allow network traffic from the nodes to the `Shoot`'s control plane, you can annotate the `Shoot` with the `shoot.gardener.cloud/skip-readiness` annotation so that the `Restore` operation finishes, and then use the [`shoots/binding`](../concepts/scheduler.md#shootsbinding-subresource) subresource to migrate the control plane back to the `Source Seed`.
+
+
+## Copying ETCD Backups Manually During the `Restore` Operation
+
+Following is a workaround that can be used to copy etcd backups manually in situations where a `Shoot`'s control plane has been moved to a `Destination Seed` and the pods running in it lack network connectivity to the `Source Seed`s storage provider:
+
+1. Follow the instructions in the [`etcd-backup-restore` getting started documentation](https://github.com/gardener/etcd-backup-restore/blob/master/docs/deployment/getting_started.md#getting-started) on how to run the `etcdbrctl` command locally or in a container.
+1. Follow the instructions in the [passing-credentials guide](https://github.com/gardener/etcd-backup-restore/blob/master/docs/deployment/getting_started.md#passing-credentials) on how to set up the required credentials for the copy operation depending on the storage providers for which you want to perform it.
+1. Use the `etcdbrctl copy` command to copy the backups by following the instructions in the [`etcdbrctl copy` guide](https://github.com/gardener/etcd-backup-restore/blob/master/docs/deployment/getting_started.md#etcdbrctl-copy)
+1. After you have successfully copied the etcd backups, wait for the `EtcdCopyBackupsTask` custom resource to be created in the `Shoot`'s control plane on the `Destination Seed`, if it does not already exist. Afterwards, mark it as successful by patching it using the following command:
+    ```
+    SHOOT_NAME=my-shoot
+    PROJECT_NAME=my-project
+
+    kubectl patch -n shoot--${PROJECT_NAME}--${SHOOT_NAME} etcdcopybackupstask ${SHOOT_NAME} --subresource status --type merge -p "{\"status\":{\"conditions\":[{\"type\":\"Succeeded\",\"status\":\"True\",\"reason\":\"manual copy successful\",\"message\":\"manual copy successful\",\"lastTransitionTime\":\"$(date -Iseconds)\",\"lastUpdateTime\":\"$(date -Iseconds)\"}]}}"
+    ```
+1. After the `main-etcd` becomes `Ready`, remove the finalizer on the source `extensions.gardener.cloud/v1alpha1.BackupEntry` (the resource has `source-` as a prefix in its name) in the `Destination Seed` so that it can be deleted successfully. This is necessary as the `Destination Seed` will not have network connectivity to perform the actual deletion of the source backup directory.
```

Comment:
> This is necessary as the `Destination Seed` will not have network connectivity to perform the actual deletion of the source backup directory.

In the next step this cleanup ought to happen manually. Doesn't this lead to the proper deletion of the source `BackupEntry`, without removing the finalizer?

Reply:
I think that the deletion of the `BackupEntry` resource as a whole would fail because the client won't be able to connect to the storage provider to do the deletion (e.g. for the aws provider - https://github.com/gardener/gardener-extension-provider-aws/blob/cb2d32524766e416dd11644ceb1b631389bb254e/pkg/controller/backupentry/actuator.go#L40-L45). This should be the case even if the backups were manually removed. If we delete the finalizer of the `extensions.BackupEntry` then.

However, I will add one more clarification to ensure that the `source-etcd-backup` has been removed (automatically) before removing the finalizer.